### PR TITLE
[ADD] *: moved the website pages from demo to data

### PR DESCRIPTION
**Pull request by dhrs-odoo**
In this PR we have moved the useful/important pages for the business from
demo to data.

Task-3761885

### CODE REVIEW

File: photography/__manifest__.py [hunk 1]
```diff
@@ -17,6 +17,7 @@
         'knowledge',
         'project_enterprise',
         'website_appointment',
+        'web_studio',
```
Comment (vava-odoo) [inline photography/__manifest__.py:20]: Hmm, not sure we want that for a web page. We might need to change the condition. Could you update the test to handle this case? (only a ir.ui.view for the website should not require studio)

File: solar_installation/data/website_menu.xml [hunk 2]
```diff
@@ -10,4 +10,10 @@
                 ('url', '=', '/default-main-menu'),
         ]).id"/>
     </record>
+    <function model="website.menu" name="unlink">
+        <value model="website.menu" eval="obj().search([
+            ('website_id', '=', ref('website.default_website')),
+            ('url', '=', '/contactus'),
+        ]).id"/>
+    </function>
```
Comment (vava-odoo) [inline solar_installation/data/website_menu.xml:18]: I think we should handle carefully existing dbs for now and avoid unlinking pages. Maybe only unpublish it in data? (and unlink in demo)

**Review by vava-odoo — COMMENTED** [review]
Thanks @dhrs-odoo for this nice work. I didn't check for other pages to move but I have two comments before moving on.

File: corporate_gifts/demo/website_theme_apply.xml [hunk 3]
```diff
@@ -31,11 +31,4 @@
         <value model="ir.ui.view" eval="obj().env['website'].with_context(website_id=obj().env.ref('website.default_website').id).viewref('website.contactus').id"/>
         <value model="ir.ui.view" eval="{'arch': obj().env.ref('corporate_gifts.contactus').arch}"/>
     </function>
-    <function model="website.menu" name="write">
-        <value model="website.menu" eval="obj().search([
-            ('website_id', '=', ref('website.default_website')),
-            ('url', '=', '/contactus'),
-        ]).id"/>
-        <value eval="{'name': 'Request a Quote'}"/>
-    </function>
```
Comment (vava-odoo) [inline corporate_gifts/demo/website_theme_apply.xml:40]: Why do you remove this?

Comment (dhrs-odoo) [reply corporate_gifts/demo/website_theme_apply.xml:40]: Sorry I forgot to add it in data file :disappointed:

File: solar_installation/demo/website_menu.xml [hunk 4]
```diff
@@ -1,13 +1,9 @@
...
-                ('website_id', '=', ref('website.default_website')),
-                ('url', '=', '/default-main-menu'),
+<odoo>
+    <function model="website.menu" name="unlink">
+        <value model="website.menu" eval="obj().search([
+            ('website_id', '=', ref('website.default_website')),
+            ('url', '=', '/contactus'),
         ]).id"/>
-    </record>
+    </function>
```
Comment (vava-odoo) [inline solar_installation/demo/website_menu.xml:8]: Unnecessary diff, keep it in website_theme_apply as before

File: solar_installation/data/website_page.xml [hunk 5]
```diff
@@ -6,4 +6,10 @@
         <field name="url">/get-a-quote</field>
         <field name="website_id" ref="website.default_website"/>
     </record>
+    <record id="unpublish_contact_us_page" model="website.page">
+        <field name="is_published" eval="False"/>
```
Comment (vava-odoo) [inline solar_installation/data/website_page.xml:10]: This doesn't seem to be sufficient, as the button is still there, leading to a 404.

File: corporate_gifts/data/website_theme_apply.xml
Comment (vava-odoo) [inline corporate_gifts/data/website_theme_apply.xml:1]: Maybe rename it to website_menu.xml?

**Review by vava-odoo — COMMENTED** [review]
Good! Just a few more comments, and I think we can merge this.

File: hardware_shop/demo/website_view.xml [hunk 6]
```diff
@@ -147,157 +147,4 @@ household functions such as cooking, cleaning and food preservation.&amp;nbsp;</
             </t>
         </field>
     </record>
-    <record id="contactus" model="ir.ui.view">
```
[the remark this answers is not in the document]
Comment (dhrs-odoo) [reply hardware_shop/demo/website_view.xml:150]: There is no need of it because CRM app is not there only so there is no point of the contactus form

**Review by vava-odoo — APPROVED** [review]
